modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/object/definitions/portlet/action/test/ObjectDefinitionExportImportTest.java

```diff
@@ -208,6 +210,23 @@ public void testExportImportObjectDefinition() throws Exception {
 		Assert.assertTrue(
 			accountRestrictedObjectDefinition.getAccountEntryRestricted());
 
+		ObjectDefinition accountObjectDefinition =
+			objectDefinitionResource.getObjectDefinitionByExternalReferenceCode(
+				"L_ACCOUNT");
+
+		ObjectRelationship objectRelationship =
+			accountObjectDefinition.getObjectRelationships()[0];
+
+		ObjectField objectField =
+			accountRestrictedObjectDefinition.getObjectFields()[7];
```

Comment:
I dont think it is safe to get `[7]` field.
Can you get using the methods from `ObjectFieldLocalService/ObjectFieldPersistence`?